src/ucm/cuda/cudamem.c

```diff
@@ -368,15 +368,15 @@ static ucs_status_t ucm_cudamem_install(int events)
 static int ucm_cudamem_scan_regions_cb(void *arg, void *addr, size_t length,
                                        int prot, const char *path)
 {
-    static const char *cuda_path_pattern = "/dev/nvidia";
-    ucm_event_handler_t *handler         = arg;
+    static const char cuda_path_pattern[] = "/dev/nvidia";
+    ucm_event_handler_t *handler          = arg;
```

Comment:
why is this change needed?

Reply:
This is a micro-optimisation that I've made while investigating the usage of `ucm_parse_proc_self_maps` callbacks. Basically instead of calculating the length of the string we just use compile time constant